contracts/registry/src/lib.rs

```diff
@@ -769,7 +769,6 @@ mod tests {
         assert_eq!(bs[1].0.owner, bob(), "bob must be second in the iterator");
         assert_eq!(bs[1].1, 103, "alice must be first in the iterator");
     }
-
```

Comment:
why removing? I think we should have empty line between functions

Reply:
DONE